.eslintrc.js

```diff
@@ -103,7 +103,12 @@ module.exports = {
         "import/prefer-default-export": "off",
         "react/state-in-constructor": "off",
         "import/extensions": "off",
-        "import/no-unresolved": "off"
+        "import/no-unresolved": "off",
+        "semi": "off",
+        "arrow-body-style": "off",
+        "@typescript-eslint/member-delimiter-style": "off",
+        "@typescript-eslint/no-use-before-define": "off",
+        "import/order": "off",
```

Comment:
why do we need to disable all these rules?

Reply:
In my opinion they offer nothing except friction for particular 'styles' of coding. In fact, some of the rules (arrow-body-style, no-use-before-define) promote less maintainable/unreadable code which I think we should avoid